src/pp_metrics.erl

```diff
@@ -9,17 +9,6 @@
 -define(METRICS_OFFER_COUNT, packet_purchaser_offer_count).
 -define(METRICS_PACKET_COUNT, packet_purchaser_packet_count).
 -define(METRICS_PACKET_DOWN_COUNT, packet_purchaser_packet_down_count).
--define(METRICS_GWMP_COUNT, packet_purchaser_gwmp_counter).
```

Comment:
We don't need that count anymore?

Reply:
It's not the number of connections. 
It was the stats of hit and missed `ack`s in the semtech protocol.